modules/javafx.controls/src/main/java/javafx/scene/control/skin/LabeledSkinBase.java

```diff
@@ -272,6 +283,7 @@ protected void updateChildren() {
                 getChildren().setAll(text);
             }
         } else {
+            // FIXME: this listener must be removed in dispose!
```

Comment:
minor: the FIXME note should be removed.

Reply:
thanks for spotting it :)